Dangerfile

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+require 'active_support/core_ext/string/output_safety'
+
+def verifyPRTitle()
+    title = github.pr_title
+    if title.size < 17
+        warn("Title too short. It should fit '[KLTB002-JIRA_TICKET_NO] TITLE' template.")
+    else 
+        if title.start_with?("[KLTB002-")
+            jiraTicketNumber = title[9, 5]
+            trailingBracket = title[14, 2]
```

Comment:
If you use the configured project code or not checking only for the title start, you will have to refactor this indexes.